src/main/resources/application.conf

```diff
@@ -550,7 +550,36 @@ mantis {
 
       # Force checksum verification of all data that is read from the file system on behalf of a particular read
       verify-checksums = true
+
+      # Use fsync on write.
+      # fsync = true
+
+      # Native LevelDB (via JNI) or LevelDB Java port.
+      native = true
+    }
+
+    rocksdb {
+      # RocksDB data directory
+      path = ${mantis.datadir}"/rocksdb/"
+
+      # Create DB data directory if it's missing
+      create-if-missing = true
+
+      # Should the DB raise an error as soon as it detects an internal corruption
+      paranoid-checks = true
+
+      # This ensures that only one thread will be occupied
+      max-threads = 1
+
+      # This ensures that only 32 open files can be accessed at once
+      max-open-files = 32
+
+      # Force checksum verification of all data that is read from the file system on behalf of a particular read
+      verify-checksums = true
     }
+
+    # Define which database to use [iodb, leveldb, rocksdb]
+    data-source = "rocksdb"
```

Comment:
I like the idea of supporting choosing db but it maybe overkill (expecially as iodb is no longer maintained and used). Actually i think it may be better to support only rocksdb, and fallback to java port of leveldb only went platform does not support rocks db.  wdyt ?